README.md

```diff
@@ -1,3 +1,14 @@
+# WARNING
+without complete rewrite this plugin CANNOT be used beyond nexus 3.77.x.    
+
+Reason: **complete BWC Break in OSS NEXUS 3.78.0 (March 2025)**:        
+"Breaking change for custom plugins: Nexus Repository migrates to Spring Boot architecture.
+Custom OSGi bundle deployment no longer supported."    
+Links:
+1. [3-78-0-release-notes](https://help.sonatype.com/en/sonatype-nexus-repository-3-78-0-release-notes.html#breaking-change-for-custom-plugins--nexus-repository-migrates-to-spring-boot-architecture)    
+2. [removed/sunsetted osgi-bundle-capability](https://help.sonatype.com/en/sonatype-nexus-repository-3-feature-status.html#osgi-bundle-capability )  
```

Comment:
@ReSearchITEng, thanks for bringing this up!

@madpah, could you verify all of the specific details here?

Then we should standardize the language and reformat this into a markdown warning using `> [!WARNING]` instead of adding a second H1.

> [!WARNING]
> Here is an example